### ISSUE TITLE
Loosen the version constraint of dash-table to avoid dependency conflicts

### ISSUE THREAD

**Issue by NeolithEra** [plotly/dash#1064]
Hi, _**dash**_ locked the version constraint of _**dash-table**_ as dash-table ==4.5.1, which leads a troubling scenario to its direct downstream project _**beeflow**_ which has dependency dash-table.

What makes the situation worse is that the downstream projects [wiley-dash-boilerplate2, metacell, clsify]  of _**dash**_ are also have dependency dash-table.

Could you please loosen the version constraint of _**dash-table**_?
Benefit of this is that users using both of _**dash**_ and _**dash-table**_ can upgrade their third party libraries in a timely manner to reduce technical debts.

### Solution

The dependency trees of your project and affected downstream projects are shown as follows.
Taking the version constraints of upstream and downstream projects into comprehensive consideration, you can

1.	Loosen click to be dash-table >=4.5.1.

2.	Try to add an upper bound for dash-table’s version constraint, according to your compatibility.

@chriddyp   Please let me know your choice. I can submit a PR to fix this issue.

Thanks for your attention.
Best,
Neolith

**Comment by NeolithEra** [comment]
### Dependency tree-----
```
dash – 1.7.0
| +- dash-core-components(install version: 1.6.0 version range:==1.6.0)
| +- dash-html-components(install version: 1.0.2 version range:==1.0.2)
| +- dash-renderer(install version: 1.2.2 version range:==1.2.2)
| +- dash-table(install version:4.5.1 version range:==4.5.1)
| +- flask(install version:1.1.1 version range:>=1.0.2)
| | +- click(install version:7.0 version range:>=5.1)
| | +- itsdangerous(install version:1.1.0 version range:>=0.24)
| | +- jinja2(install version:2.10.3 version range:>=2.10.1)
| | | +- markupsafe(install version:1.1.1 version range:>=0.23)
| | +- werkzeug(install version:0.16.0 version range:>=0.15)
| +- flask-compress(install version:1.4.0 version range:*)
| +- plotly(install version:4.3.0 version range:*)
| | +- retrying(install version:1.3.3 version range:>=1.3.3)
| | | +- six(install version:1.13.0 version range:>=1.7.0)
| | +- six(install version:1.13.0 version range:*)
```
```
beeflow - 0.0.2
| +- dash(install version:1.6.1 version range:*)
| +- dash-core-components(install version:1.6.0 version range:*)
| +- dash-html-components(install version:1.0.2 version range:*)
| +- dash-resumable-upload(install version:0.0.3 version range:*)
| +- dash-table(install version:4.5.1 version range:*)
| +- pandas(install version:0.25.3 version range:*)
| +- plotly(install version:4.3.0 version range:*)
| | +- retrying(install version:1.3.3 version range:>=1.3.3)
| | | +- six(install version:1.13.0 version range:>=1.7.0)
| | +- six(install version:1.13.0 version range:*) ```
```

```
wiley-dash-boilerplate2 - master
| +- boto3(install version:1.10.23 version range:*)
| | +- botocore(install version:1.13.46 version range:>=1.13.23,<1.14.0)
| | +- jmespath(install version:0.9.4 version range:<1.0.0,>=0.7.1)
| | +- s3transfer(install version:0.2.1 version range:>=0.2.0,<0.3.0)
| | | +- botocore(install version:1.13.46 version range:<2.0.0,>=1.12.36)
| +- celery(install version:4.4.0rc4 version range:*)
| +- dash(install version:1.6.1 version range:*)
| +- dash-table(install version:4.5.1 version range:*)
| +- gunicorn(install version:20.0.2 version range:*)
| | +- setuptools(install version:42.0.1 version range:>=3.0)
| +- pipdate(install version:0.3.4 version range:*)
| | +- appdirs(install version:1.4.3 version range:*)
| | +- requests(install version:2.22.0 version range:*)
| | | +- certifi(install version:2019.9.11 version range:>=2017.4.17)
| | | +- chardet(install version:3.0.4 version range:<3.1.0,>=3.0.2)
| | | +- idna(install version:2.8 version range:>=2.5,<2.9)
| | | +- urllib3(install version:1.25.7 version range:<1.26,>=1.21.1)
| +- redis(install version:3.3.11 version range:*)
| +- requests-aws4auth(install version:0.9 version range:*)
| | +- requests(install version:2.22.0 version range:*)
| | | +- certifi(install version:2019.9.11 version range:>=2017.4.17)
| | | +- chardet(install version:3.0.4 version range:<3.1.0,>=3.0.2)
| | | +- idna(install version:2.8 version range:>=2.5,<2.9)
| | | +- urllib3(install version:1.25.7 version range:<1.26,>=1.21.1)
| +- s3fs(install version:0.4.0 version range:*)
| | +- boto3(install version:1.10.23 version range:>=1.9.91)
| | | +- botocore(install version:1.13.46 version range:>=1.13.23,<1.14.0)
| | | +- jmespath(install version:0.9.4 version range:<1.0.0,>=0.7.1)
| | | +- s3transfer(install version:0.2.1 version range:>=0.2.0,<0.3.0)
| | | | +- botocore(install version:1.13.46 version range:<2.0.0,>=1.12.36)
| | +- botocore(install version:1.13.46 version range:>=1.12.91)
| | +- fsspec(install version:0.6.0 version range:>=0.6.0)
| +- xlrd(install version:1.2.0 version range:*)
| +- xlsxwriter(install version:1.2.6 version range:*)
```

```
metacell - 0.5.44
| +- dash(install version:1.6.1 version range:*)
| +- dash-core-components(install version:1.6.0 version range:*)
| +- dash-html-components(install version:1.0.2 version range:*)
| +- dash-table(install version:4.5.1 version range:*)
| +- dynamake(install version:0.4.25 version range:*)
| | +- prwlock(install version:0.4.0 version range:*)
| | +- pyyaml(install version:5.2b1 version range:*)
| | +- sortedcontainers(install version:2.1.0 version range:*)
| | +- termcolor(install version:1.1.0 version range:*)
| +- feather-format(install version:0.4.0 version range:*)
| | +- pyarrow(install version:0.15.1 version range:>=0.4.0)
| | | +- enum34(install version:1.1.6 version range:>=1.1.6)
| | | +- futures(install version:3.3.0 version range:*)
| | | +- numpy(install version:1.17.4 version range:>=1.14)
| | | +- six(install version:1.13.0 version range:>=1.0.0)
| +- flask(install version:1.1.1 version range:*)
| | +- click(install version:7.0 version range:>=5.1)
| | +- itsdangerous(install version:1.1.0 version range:>=0.24)
| | +- jinja2(install version:2.10.3 version range:>=2.10.1)
| | | +- markupsafe(install version:1.1.1 version range:>=0.23)
| | +- werkzeug(install version:0.16.0 version range:>=0.15)
| +- h5py(install version:2.10.0 version range:*)
| +- numpy(install version:1.17.4 version range:*)
| +- pandas(install version:0.25.3 version range:*)
| +- plotly(install version:4.3.0 version range:*)
| | +- retrying(install version:1.3.3 version range:>=1.3.3)
| | | +- six(install version:1.13.0 version range:>=1.7.0)
| | +- six(install version:1.13.0 version range:*)
| +- scipy(install version:1.3.3 version range:*)
| +- tgutils(install version:0.2.28 version range:*)
| | +- dynamake(install version:0.4.25 version range:*)
| | | +- prwlock(install version:0.4.0 version range:*)
| | | +- pyyaml(install version:5.2b1 version range:*)
| | | +- sortedcontainers(install version:2.1.0 version range:*)
| | | +- termcolor(install version:1.1.0 version range:*)
| | +- numpy(install version:1.17.4 version range:*)
| | +- pandas(install version:0.25.3 version range:*)
| | +- pyyaml(install version:5.2b1 version range:*)
```

```
clsify - 0.1.1
| +- attr(install version:0.3.1 version range:*)
| +- bioconvert(install version:0.4.3 version range:*)
| | +- bamtools(install version: version range:*)
| | +- bcftools(install version:1.8 version range:==1.8)
| | +- bedops(install version: version range:*)
| | +- bedtools(install version: version range:*)
| | +- biopython(install version:1.75 version range:>=1.70)
| | | +- numpy(install version:1.17.4 version range:*)
| | +- colorlog(install version:4.0.2 version range:*)
| | +- coveralls(install version:1.8.2 version range:*)
| | | +- coverage(install version:4.5.4 version range:<5.0,>=3.6)
| | | +- docopt(install version:0.6.2 version range:>=0.6.1)
| | | +- requests(install version:2.22.0 version range:>=1.0.0)
| | | | +- certifi(install version:2019.9.11 version range:>=2017.4.17)
| | | | +- chardet(install version:3.0.4 version range:<3.1.0,>=3.0.2)
| | | | +- idna(install version:2.8 version range:>=2.5,<2.9)
| | | | +- urllib3(install version:1.25.7 version range:<1.26,>=1.21.1)
| | +- deeptools(install version:3.3.1 version range:*)
| | | +- deeptoolsintervals(install version:0.1.9 version range:>=0.1.8)
| | | +- matplotlib(install version:3.1.2 version range:>=3.0.0)
| | | +- numpy(install version:1.17.4 version range:>=1.9.0)
| | | +- numpydoc(install version:0.9.1 version range:>=0.5)
| | | | +- jinja2(install version:2.10.3 version range:>=2.3)
| | | | +- sphinx(install version:2.2.1 version range:>=1.6.5)
| | | +- plotly(install version:4.3.0 version range:>=2.0.0)
| | | | +- retrying(install version:1.3.3 version range:>=1.3.3)
| | | | +- six(install version:1.13.0 version range:*)
| | | +- py2bit(install version:0.3.0 version range:>=0.2.0)
| | | +- pybigwig(install version:0.3.17 version range:>=0.2.1)
| | | +- pysam(install version:0.15.3 version range:>=0.14.0)
| | | | +- cython(install version:0.29.14 version range:>=0.24.1)
| | | +- scipy(install version:1.3.3 version range:>=0.17.0)
| | +- dsrc(install version: version range:*)
| | +- easydev(install version:0.9.38 version range:*)
| | | +- colorama(install version:0.4.1 version range:*)
| | | +- colorlog(install version:4.0.2 version range:*)
| | | +- pexpect(install version:4.7.0 version range:*)
| | | | +- ptyprocess(install version:0.6.0 version range:>=0.5)
| | +- mappy(install version:2.17 version range:*)
| | +- matplotlib(install version:3.1.2 version range:*)
| | +- mawk(install version: version range:*)
| | +- mock(install version:3.0.5 version range:*)
| | +- mosdepth(install version: version range:*)
| | +- networkx(install version:2.4 version range:*)
| | | +- decorator(install version:4.4.1 version range:>=4.3.0)
| | +- numpydoc(install version:0.9.1 version range:*)
| | | +- jinja2(install version:2.10.3 version range:>=2.3)
| | | | +- markupsafe(install version:1.1.1 version range:>=0.23)
| | | +- sphinx(install version:2.2.1 version range:>=1.6.5)
| | | | +- alabaster(install version:0.7.12 version range:>=0.7,<0.8)
| | | | +- babel(install version:2.7.0 version range:>=1.3)
| | | | +- docutils(install version:0.15.2 version range:>=0.12)
| | | | +- imagesize(install version:1.1.0 version range:*)
| | | | +- jinja2(install version:2.10.3 version range:>=2.3)
| | | | +- packaging(install version:19.2 version range:*)
| | | | +- pygments(install version:2.5.1 version range:>=2.0)
| | | | +- requests(install version:2.22.0 version range:>=2.5.0)
| | | | +- setuptools(install version:42.0.1 version range:*)
| | | | +- snowballstemmer(install version:2.0.0 version range:>=1.1)
| | | | +- sphinxcontrib-applehelp(install version:1.0.1 version range:*)
| | | | +- sphinxcontrib-devhelp(install version:1.0.1 version range:*)
| | | | +- sphinxcontrib-htmlhelp(install version:1.0.2 version range:*)
| | | | +- sphinxcontrib-jsmath(install version:1.0.1 version range:*)
| | | | +- sphinxcontrib-qthelp(install version:1.0.2 version range:*)
| | | | +- sphinxcontrib-serializinghtml(install version:1.1.3 version range:*)
| | +- pandas(install version:0.25.3 version range:*)
| | +- pbzip2(install version: version range:*)
| | +- pigz(install version: version range:*)
| | +- pillow(install version:6.2.1 version range:*)
| | | +- black(install version:19.10b0 version range:*)
| | | | +- appdirs(install version:1.4.3 version range:*)
| | | | +- attrs(install version:19.3.0 version range:>=18.1.0)
| | | | +- click(install version:7.0 version range:>=6.5)
| | | | +- pathspec(install version:0.7.0 version range:<1,>=0.6)
| | | | +- regex(install version:2019.11.1 version range:*)
| | | | +- toml(install version:0.10.0 version range:>=0.9.4)
| | | | +- typed-ast(install version:1.4.0 version range:>=1.4.0)
| | | +- check-manifest(install version:0.40 version range:*)
| | | | +- toml(install version:0.10.0 version range:*)
| | | +- coverage(install version:5.0b1 version range:*)
| | | +- coveralls(install version:1.8.2 version range:*)
| | | | +- coverage(install version:4.5.4 version range:<5.0,>=3.6)
| | | | +- docopt(install version:0.6.2 version range:>=0.6.1)
| | | | +- requests(install version:2.22.0 version range:>=1.0.0)
| | | +- jarn.viewdoc(install version:2.3 version range:*)
| | | +- olefile(install version:0.46 version range:*)
| | | +- pycodestyle(install version:2.5.0 version range:*)
| | | +- pyflakes(install version:2.1.1 version range:*)
| | | +- pyroma(install version:2.6 version range:*)
| | | | +- docutils(install version:0.15.2 version range:*)
| | | | +- pygments(install version:2.5.1 version range:*)
| | | | +- setuptools(install version:42.0.1 version range:*)
| | | +- pytest(install version:5.3.1 version range:*)
| | | | +- atomicwrites(install version:1.3.0 version range:>=1.0)
| | | | +- attrs(install version:19.3.0 version range:>=17.4.0)
| | | | +- colorama(install version:0.4.1 version range:*)
| | | | +- importlib-metadata(install version:0.23 version range:>=0.12)
| | | | +- more-itertools(install version:7.2.0 version range:>=4.0.0)
| | | | +- packaging(install version:19.2 version range:*)
| | | | +- pathlib2(install version:2.3.5 version range:>=2.2.0)
| | | | +- pluggy(install version:0.13.1 version range:>=0.12,<1.0)
| | | | +- py(install version:1.8.0 version range:>=1.5.0)
| | | | +- wcwidth(install version:0.1.7 version range:*)
| | | +- pytest-cov(install version:2.8.1 version range:*)
| | | | +- coverage(install version:5.0b1 version range:>=4.4)
| | | | +- pytest(install version:5.3.1 version range:>=3.6)
| | | +- sphinx-rtd-theme(install version:0.4.3 version range:*)
| | | | +- sphinx(install version:2.2.1 version range:*)
| | +- plink(install version:2.3 version range:*)
| | | +- future(install version:0.18.2 version range:*)
| | +- py2bit(install version:0.3.0 version range:*)
| | +- pybigwig(install version:0.3.17 version range:*)
| | +- pyexcel(install version:0.5.15 version range:*)
| | | +- lml(install version:0.0.9 version range:>=0.0.4)
| | | +- pyexcel-io(install version:0.5.20 version range:>=0.5.19)
| | | | +- lml(install version:0.0.9 version range:>=0.0.4)
| | +- pyexcel-ods3(install version:0.5.3 version range:*)
| | | +- lxml(install version:4.4.1 version range:*)
| | | | +- cython(install version:0.29.14 version range:>=0.29.7)
| | | +- pyexcel-ezodf(install version:0.3.4 version range:>=0.3.3)
| | | | +- lxml(install version:4.4.1 version range:*)
| | | +- pyexcel-io(install version:0.5.20 version range:>=0.5.10)
| | | | +- lml(install version:0.0.9 version range:>=0.0.4)
| | +- pyexcel-xls(install version:0.5.8 version range:*)
| | | +- pyexcel-io(install version:0.5.20 version range:>=0.5.3)
| | | | +- lml(install version:0.0.9 version range:>=0.0.4)
| | | +- xlrd(install version:1.2.0 version range:*)
| | | +- xlwt(install version:1.3.0 version range:*)
| | | | +- coveralls(install version:1.8.2 version range:*)
| | | | +- nose(install version:1.3.7 version range:*)
| | | | +- nose-cov(install version:1.6 version range:*)
| | | | +- nose-fixes(install version:1.3 version range:*)
| | | | +- panci(install version:0.0.2 version range:*)
| | | | +- pkginfo(install version:1.5.0.1 version range:*)
| | | | +- setuptools-git(install version:1.2 version range:*)
| | | | +- six(install version:1.13.0 version range:*)
| | | | +- sphinx(install version:2.2.1 version range:*)
| | | | +- tox(install version:3.14.0 version range:*)
| | | | +- twine(install version:3.1.1 version range:*)
| | | | +- wheel(install version:0.33.6 version range:*)
| | +- pygraphviz(install version:1.5 version range:*)
| | +- pysam(install version:0.15.3 version range:*)
| | | +- cython(install version:0.29.14 version range:>=0.24.1)
| | +- pytest(install version:5.3.1 version range:*)
| | | +- atomicwrites(install version:1.3.0 version range:>=1.0)
| | | +- attrs(install version:19.3.0 version range:>=17.4.0)
| | | +- colorama(install version:0.4.1 version range:*)
| | | +- importlib-metadata(install version:0.23 version range:>=0.12)
| | | +- more-itertools(install version:7.2.0 version range:>=4.0.0)
| | | +- packaging(install version:19.2 version range:*)
| | | | +- pyparsing(install version:2.4.5 version range:>=2.0.2)
| | | | +- six(install version:1.13.0 version range:*)
| | | +- pathlib2(install version:2.3.5 version range:>=2.2.0)
| | | | +- six(install version:1.13.0 version range:*)
| | | +- pluggy(install version:0.13.1 version range:>=0.12,<1.0)
| | | | +- importlib-metadata(install version:0.23 version range:>=0.12)
| | | +- py(install version:1.8.0 version range:>=1.5.0)
| | | +- wcwidth(install version:0.1.7 version range:*)
| | +- pytest-cov(install version:2.8.1 version range:*)
| | | +- coverage(install version:5.0b1 version range:>=4.4)
| | | +- pytest(install version:5.3.1 version range:>=3.6)
| | | | +- atomicwrites(install version:1.3.0 version range:>=1.0)
| | | | +- attrs(install version:19.3.0 version range:>=17.4.0)
| | | | +- colorama(install version:0.4.1 version range:*)
| | | | +- importlib-metadata(install version:0.23 version range:>=0.12)
| | | | +- more-itertools(install version:7.2.0 version range:>=4.0.0)
| | | | +- packaging(install version:19.2 version range:*)
| | | | +- pathlib2(install version:2.3.5 version range:>=2.2.0)
| | | | +- pluggy(install version:0.13.1 version range:>=0.12,<1.0)
| | | | +- py(install version:1.8.0 version range:>=1.5.0)
| | | | +- wcwidth(install version:0.1.7 version range:*)
| | +- pytest-env(install version:0.6.2 version range:*)
| | | +- pytest(install version:5.3.1 version range:>=2.6.0)
| | | | +- atomicwrites(install version:1.3.0 version range:>=1.0)
| | | | +- attrs(install version:19.3.0 version range:>=17.4.0)
| | | | +- colorama(install version:0.4.1 version range:*)
| | | | +- importlib-metadata(install version:0.23 version range:>=0.12)
| | | | +- more-itertools(install version:7.2.0 version range:>=4.0.0)
| | | | +- packaging(install version:19.2 version range:*)
| | | | +- pathlib2(install version:2.3.5 version range:>=2.2.0)
| | | | +- pluggy(install version:0.13.1 version range:>=0.12,<1.0)
| | | | +- py(install version:1.8.0 version range:>=1.5.0)
| | | | +- wcwidth(install version:0.1.7 version range:*)
| | +- pytest-mock(install version:1.12.1 version range:*)
| | | +- mock(install version:3.0.5 version range:*)
| | | +- pytest(install version:5.3.1 version range:>=2.7)
| | | | +- atomicwrites(install version:1.3.0 version range:>=1.0)
| | | | +- attrs(install version:19.3.0 version range:>=17.4.0)
| | | | +- colorama(install version:0.4.1 version range:*)
| | | | +- importlib-metadata(install version:0.23 version range:>=0.12)
| | | | +- more-itertools(install version:7.2.0 version range:>=4.0.0)
| | | | +- packaging(install version:19.2 version range:*)
| | | | +- pathlib2(install version:2.3.5 version range:>=2.2.0)
| | | | +- pluggy(install version:0.13.1 version range:>=0.12,<1.0)
| | | | +- py(install version:1.8.0 version range:>=1.5.0)
| | | | +- wcwidth(install version:0.1.7 version range:*)
| | +- pytest-runner(install version:5.2 version range:*)
| | +- pytest-timeout(install version:1.3.3 version range:*)
| | | +- pytest(install version:5.3.1 version range:>=3.6.0)
| | | | +- atomicwrites(install version:1.3.0 version range:>=1.0)
| | | | +- attrs(install version:19.3.0 version range:>=17.4.0)
| | | | +- colorama(install version:0.4.1 version range:*)
| | | | +- importlib-metadata(install version:0.23 version range:>=0.12)
| | | | +- more-itertools(install version:7.2.0 version range:>=4.0.0)
| | | | +- packaging(install version:19.2 version range:*)
| | | | +- pathlib2(install version:2.3.5 version range:>=2.2.0)
| | | | +- pluggy(install version:0.13.1 version range:>=0.12,<1.0)
| | | | +- py(install version:1.8.0 version range:>=1.5.0)
| | | | +- wcwidth(install version:0.1.7 version range:*)
| | +- pytest-xdist(install version:1.30.0 version range:*)
| | | +- execnet(install version:1.7.1 version range:>=1.1)
| | | | +- apipkg(install version:1.5 version range:>=1.4)
| | | +- pytest(install version:5.3.1 version range:>=4.4.0)
| | | | +- atomicwrites(install version:1.3.0 version range:>=1.0)
| | | | +- attrs(install version:19.3.0 version range:>=17.4.0)
| | | | +- colorama(install version:0.4.1 version range:*)
| | | | +- importlib-metadata(install version:0.23 version range:>=0.12)
| | | | +- more-itertools(install version:7.2.0 version range:>=4.0.0)
| | | | +- packaging(install version:19.2 version range:*)
| | | | +- pathlib2(install version:2.3.5 version range:>=2.2.0)
| | | | +- pluggy(install version:0.13.1 version range:>=0.12,<1.0)
| | | | +- py(install version:1.8.0 version range:>=1.5.0)
| | | | +- wcwidth(install version:0.1.7 version range:*)
| | | +- pytest-forked(install version:1.1.3 version range:*)
| | | | +- pytest(install version:5.3.1 version range:>=3.1.0)
| | | +- six(install version:1.13.0 version range:*)
| | +- pyyaml(install version:5.2b1 version range:*)
| | +- sambamba(install version: version range:*)
| | +- samtools(install version: version range:>=1.9)
| | +- sed(install version:0.3.1 version range:*)
| | +- seqtk(install version: version range:*)
| | +- sphinx(install version:2.2.1 version range:*)
| | | +- alabaster(install version:0.7.12 version range:>=0.7,<0.8)
| | | +- babel(install version:2.7.0 version range:>=1.3)
| | | | +- pytz(install version:2019.3 version range:>=0a)
| | | +- docutils(install version:0.15.2 version range:>=0.12)
| | | +- imagesize(install version:1.1.0 version range:*)
| | | +- jinja2(install version:2.10.3 version range:>=2.3)
| | | | +- markupsafe(install version:1.1.1 version range:>=0.23)
| | | +- packaging(install version:19.2 version range:*)
| | | | +- pyparsing(install version:2.4.5 version range:>=2.0.2)
| | | | +- six(install version:1.13.0 version range:*)
| | | +- pygments(install version:2.5.1 version range:>=2.0)
| | | +- requests(install version:2.22.0 version range:>=2.5.0)
| | | | +- certifi(install version:2019.9.11 version range:>=2017.4.17)
| | | | +- chardet(install version:3.0.4 version range:<3.1.0,>=3.0.2)
| | | | +- idna(install version:2.8 version range:>=2.5,<2.9)
| | | | +- urllib3(install version:1.25.7 version range:<1.26,>=1.21.1)
| | | +- setuptools(install version:42.0.1 version range:*)
| | | +- snowballstemmer(install version:2.0.0 version range:>=1.1)
| | | +- sphinxcontrib-applehelp(install version:1.0.1 version range:*)
| | | +- sphinxcontrib-devhelp(install version:1.0.1 version range:*)
| | | +- sphinxcontrib-htmlhelp(install version:1.0.2 version range:*)
| | | +- sphinxcontrib-jsmath(install version:1.0.1 version range:*)
| | | +- sphinxcontrib-qthelp(install version:1.0.2 version range:*)
| | | +- sphinxcontrib-serializinghtml(install version:1.1.3 version range:*)
| | +- sphinx-gallery(install version:0.5.0 version range:*)
| | | +- matplotlib(install version:3.1.2 version range:*)
| | | +- pillow(install version:6.2.1 version range:*)
| | | | +- black(install version:19.10b0 version range:*)
| | | | +- check-manifest(install version:0.40 version range:*)
| | | | +- coverage(install version:5.0b1 version range:*)
| | | | +- coveralls(install version:1.8.2 version range:*)
| | | | +- jarn.viewdoc(install version:2.3 version range:*)
| | | | +- olefile(install version:0.46 version range:*)
| | | | +- pycodestyle(install version:2.5.0 version range:*)
| | | | +- pyflakes(install version:2.1.1 version range:*)
| | | | +- pyroma(install version:2.6 version range:*)
| | | | +- pytest(install version:5.3.1 version range:*)
| | | | +- pytest-cov(install version:2.8.1 version range:*)
| | | | +- sphinx-rtd-theme(install version:0.4.3 version range:*)
| | | +- setuptools(install version:42.0.1 version range:*)
| | | +- sphinx(install version:2.2.1 version range:>=1.8.3)
| | | | +- alabaster(install version:0.7.12 version range:>=0.7,<0.8)
| | | | +- babel(install version:2.7.0 version range:>=1.3)
| | | | +- docutils(install version:0.15.2 version range:>=0.12)
| | | | +- imagesize(install version:1.1.0 version range:*)
| | | | +- jinja2(install version:2.10.3 version range:>=2.3)
| | | | +- packaging(install version:19.2 version range:*)
| | | | +- pygments(install version:2.5.1 version range:>=2.0)
| | | | +- requests(install version:2.22.0 version range:>=2.5.0)
| | | | +- setuptools(install version:42.0.1 version range:*)
| | | | +- snowballstemmer(install version:2.0.0 version range:>=1.1)
| | | | +- sphinxcontrib-applehelp(install version:1.0.1 version range:*)
| | | | +- sphinxcontrib-devhelp(install version:1.0.1 version range:*)
| | | | +- sphinxcontrib-htmlhelp(install version:1.0.2 version range:*)
| | | | +- sphinxcontrib-jsmath(install version:1.0.1 version range:*)
| | | | +- sphinxcontrib-qthelp(install version:1.0.2 version range:*)
| | | | +- sphinxcontrib-serializinghtml(install version:1.1.3 version range:*)
| | +- sphinx-rtd-theme(install version:0.4.3 version range:*)
| | | +- sphinx(install version:2.2.1 version range:*)
| | | | +- alabaster(install version:0.7.12 version range:>=0.7,<0.8)
| | | | +- babel(install version:2.7.0 version range:>=1.3)
| | | | +- docutils(install version:0.15.2 version range:>=0.12)
| | | | +- imagesize(install version:1.1.0 version range:*)
| | | | +- jinja2(install version:2.10.3 version range:>=2.3)
| | | | +- packaging(install version:19.2 version range:*)
| | | | +- pygments(install version:2.5.1 version range:>=2.0)
| | | | +- requests(install version:2.22.0 version range:>=2.5.0)
| | | | +- setuptools(install version:42.0.1 version range:*)
| | | | +- snowballstemmer(install version:2.0.0 version range:>=1.1)
| | | | +- sphinxcontrib-applehelp(install version:1.0.1 version range:*)
| | | | +- sphinxcontrib-devhelp(install version:1.0.1 version range:*)
| | | | +- sphinxcontrib-htmlhelp(install version:1.0.2 version range:*)
| | | | +- sphinxcontrib-jsmath(install version:1.0.1 version range:*)
| | | | +- sphinxcontrib-qthelp(install version:1.0.2 version range:*)
| | | | +- sphinxcontrib-serializinghtml(install version:1.1.3 version range:*)
| | +- squizz(install version: version range:*)
| | +- sra-tools(install version: version range:*)
| | +- ucsc-bedgraphtobigwig(install version: version range:*)
| | +- ucsc-bigwigtobedgraph(install version: version range:*)
| | +- ucsc-fatotwobit(install version: version range:*)
| | +- ucsc-twobittofa(install version: version range:*)
| | +- wiggletools(install version: version range:>=1.2.2)
| | +- xlrd(install version:1.2.0 version range:*)
| +- cattrs(install version:1.0.0rc0 version range:*)
| +- cryptography(install version:2.8 version range:*)
| +- dash(install version:1.6.1 version range:*)
| +- dash-bootstrap-components(install version:0.7.2 version range:*)
| | +- dash(install version:1.6.1 version range:>=1.0.0)
| +- dash-core-components(install version:1.6.0 version range:*)
| +- dash-html-components(install version:1.0.2 version range:*)
| +- dash-renderer(install version:1.2.2 version range:*)
| +- dash-table(install version:4.5.1 version range:*)
| +- logzero(install version:1.5.0 version range:*)
| +- plotly(install version:4.3.0 version range:*)
| | +- retrying(install version:1.3.3 version range:>=1.3.3)
| | | +- six(install version:1.13.0 version range:>=1.7.0)
| | +- six(install version:1.13.0 version range:*)
| +- pyopenssl(install version:19.1.0 version range:*)
| | +- cryptography(install version:2.8 version range:>=2.8)
| | +- six(install version:1.13.0 version range:>=1.5.2)
| +- pysam(install version:0.15.3 version range:==0.15.3)
| | +- cython(install version:0.29.14 version range:>=0.24.1)
| +- requests(install version:2.22.0 version range:*)
| | +- certifi(install version:2019.9.11 version range:>=2017.4.17)
| | +- chardet(install version:3.0.4 version range:<3.1.0,>=3.0.2)
| | +- idna(install version:2.8 version range:>=2.5,<2.9)
| | +- urllib3(install version:1.25.7 version range:<1.26,>=1.21.1)
| +- vcfpy(install version:0.12.1 version range:*)
| +- xlsxwriter(install version:1.2.6 version range:*)
```

**Comment by alexcjohnson** [comment]
merged with #1062
